### PR TITLE
Add Covenant - Disable the dead caravan from Stockton

### PR DESCRIPTION
**Pull request by DoubleYouC**
* Add plugin to 'Fixes' section.

* Add location
  * [https://www.nexusmods.com/fallout4/mods/43961/](https://www.nexusmods.com/fallout4/mods/43961/)

* Assign 'Fixes' group
  * Edits records not likely to be altered by other mods.

* Add cleaning data

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1672,6 +1672,17 @@ plugins:
     group: *fixesGroup
     msg: [ *alreadyInUFO4P ]
 
+  - name: 'DisabletheCaravan.esl'
```
Comment (sibir-ine) [inline masterlist.yaml]: There's an ESP version on the page that'd benefit from the same group assignment:

```suggestion
  - name: 'DisabletheCaravan\.es[lp]'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -1672,6 +1672,17 @@ plugins:
...
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/43961'
```
Comment (sibir-ine) [inline masterlist.yaml]: We add trailing slashes to URLs were possible for consistency & convenience (it makes it easier to search for, e.g., Nexus page IDs):

```suggestion
      - link: 'https://www.nexusmods.com/fallout4/mods/43961/'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -1672,6 +1672,17 @@ plugins:
...
+        name: 'Covenant - Disable the dead caravan from Stockton'
+    group: *fixesGroup
+    dirty:
```
Comment (sibir-ine) [inline masterlist.yaml:1681]: To go with the name change above as this cleaning data only applies to the ESL:

```suggestion
  - name: 'DisabletheCaravan.esl'
    dirty:
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
Besides the suggestions, when submitting cleaning data, can you please say for what version (if more than one available) & what version number? I've double-checked & it appears to be v1.1.

**Comment by DoubleYouC** [comment]
> Besides the suggestions, when submitting cleaning data, can you please say for what version (if more than one available) & what version number? I've double-checked & it appears to be v1.1.

Will do.

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**